Add article on progress update

cburgdorf opened the pull request.

PascalPrecht commented on source/_posts/progress-update-on-our-machine-learning-jump-start-course.md:
```diff
@@ -0,0 +1,31 @@
+tags: announcement, course
+---
+
+Last year we [announced to build a fresh, beginner-friendly Machine Learning course](/2017/12/15/machine-learning-jump-start-online-course/) with the promise to make this exciting field more accesible for starters. Many people want to get started with Machine Learning but have yet to find a course that isn't overloaded by heavy math and science. Our Machine Learning Jump Start breaks with that by ...
```
> [...] exciting field more acces**s**ible for [...]

> Our [Machine Learning Jump Start](https://course.machinelabs.ai) breaks [...]

> [...] breaks with that by explain**in**g concepts [...]

> [...] practical online labs **using the MachineLabs platform**

PascalPrecht commented on source/_posts/progress-update-on-our-machine-learning-jump-start-course.md:
```diff
@@ -0,0 +1,31 @@
+
+<!-- more -->
+
+The course has been in [pre-sale](http://course.machinelabs.ai) since then and it's been very motivating for us to see all the excitment about the course from people all around the world.
```
How about:

> [...] and it's been very motivating for us to see all the excit**e**ment from our community all around the world.

PascalPrecht commented on source/_posts/progress-update-on-our-machine-learning-jump-start-course.md:
```diff
@@ -0,0 +1,31 @@
+
+The course has been in [pre-sale](http://course.machinelabs.ai) since then and it's been very motivating for us to see all the excitment about the course from people all around the world.
+
+The format of the course is mainly video / screencast driven but also includes interactive labs that run entirely on the MachineLabs platform.
```
Not sure what exactly is the purpose of this sentence here.

PascalPrecht commented on source/_posts/progress-update-on-our-machine-learning-jump-start-course.md:
```diff
@@ -0,0 +1,31 @@
+
+Today we like to share an update about our progress and the current timeline for the course.
+
+Before we dive into this, enjoy this little teaser video that we recorded for you at our last company offsite in Oslo earlier this year.
```
Merge this paragraph into the previous one.

PascalPrecht commented on source/_posts/progress-update-on-our-machine-learning-jump-start-course.md:
```diff
@@ -0,0 +1,31 @@
+
+INTRO_VIDEO_HERE
+
+While we consider the course a long-term effort and plan to update the courseware continuously, our initial plan was to get the Early Access Preview (EAP) out this month.
```
>  [...] and plan to update the <s>courseware</s> it continuously,

> [...] get the Early Access Preview (EAP) out this month (April 2018).

^ This last one is mainly for people who'll read the post much later and don't pay attention to the publish date of the post.

PascalPrecht commented on source/_posts/progress-update-on-our-machine-learning-jump-start-course.md:
```diff
@@ -0,0 +1,31 @@
+
+To ensure we're shipping something that people will like we've decided to take a bit more time for our EAP to mature.
+
+Our new target date to deliver the EAP is the **end of Q3 2018**. Once the EAP is out everyone can expect to receive **monthly updates** to the courseware.
```
Merge this paragraph into the previous one.

PascalPrecht commented on source/_posts/progress-update-on-our-machine-learning-jump-start-course.md:
```diff
@@ -0,0 +1,31 @@
+
+While we consider the course a long-term effort and plan to update the courseware continuously, our initial plan was to get the Early Access Preview (EAP) out this month.
+
+**Please note that no one has been or will be charged a single dollar before the EAP is out.**
```
> Please <s>note</s> remember that no one has or will [...]

This and the following paragraph should be moved to *after* we say "we're pushing it back"

cburgdorf commented:
@PascalPrecht Thanks for the feedback. Addressed all points. Maybe our youtube channel administrator @d3lm could upload the intro video so that I can include it in the post.

d3lm commented:
Done. Video is uploaded.
